gulp/utils/configHelpers.js

```diff
@@ -0,0 +1,22 @@
+const glob = require('glob');
+
+module.exports = {
+  getAvailableLanguages: (config) => {
+    const entry = config.entryGlobs.content;
+    const entryPath = config.entryPaths.content;
+    return entry.map(e => glob.sync(e))
+      .reduce((a, b) => a.concat(b), [])
+      .map((path) => {
+        const withoutBase = path.replace(entryPath, '');
```

Comment:
Can we set the entrypath as root or cwd in the options of the glob.sync so we avoid this line?